packages/detox/src/executors/test/schema.json

```diff
@@ -21,18 +21,21 @@
         "ios.sim.release",
         "android.emu.debug",
         "android.emu.release"
-      ]
+      ],
+      "x-priority": "important"
     },
     "buildTarget": {
       "type": "string",
-      "description": "Target which builds the application."
+      "description": "Target which builds the application.",
+      "x-priority": "important"
```

Comment:
according to the guidelines in asana, `buildTarget` also shouldn't be important.